cypress/integration/users.spec.js

```diff
@@ -0,0 +1,40 @@
+// users.spec.js tests User groups in settings and in user app
+
+describe('Validates Users groups', () => {
+    before(() => {
+        const userEnv = Cypress.env('ADMIN_USER')
+        const passEnv = Cypress.env('ADMIN_PASS')
+        cy.login(userEnv, passEnv)
+      })
+    
+    after(() => {
+        cy.logout()
+    })
+
+    it('Opens User Groups in settings', () => {
+        cy.pageVisit('settings/users/')
+        cy.contains('Patron groups')
+    })
+
+    it('Opens Patron groups and checks values', () => {
+        cy.contains('Patron groups').click()
+
+        cy.wait(5000)
+
+        cy.contains('Faculty-Affiliate')
+
+        cy.get("#editList-patrongroups > div > div > div").should('have.length', 27)
```

Comment:
It seems like this test would fail since there are 21 patron groups. Maybe I'm misunderstanding what `have.length` is counting. There's a `data-total-count` attribute in the same div as `id=editList-patrongroups`.